frontend/tests/unit/Feed.spec.js

```diff
@@ -29,10 +29,10 @@ describe('Feed.vue Reviews', () => {
 describe('Feed.vue Questions', () => {
   before(function() {
     this.items = [
-      { id: 1, title: 'hey', body: 'wow', likes: 2, published: new Date().toDateString() },
-      { id: 2, title: 'blurggh', body: 'wowzers', likes: 3, published: new Date().toDateString() },
-      { id: 3, title: 'blurggh', body: 'woawders', likes: 3, published: new Date().toDateString() },
-      { id: 4, title: 'blurggh', body: 'wdiajdw', likes: 3, published: new Date().toDateString() }
+      { id: '1', title: 'hey', body: 'wow', likes: 2, published: new Date().toDateString() },
```

Comment:
Maybe id should be a number I dunno. Where is it getting affected most? When it comes out of the URL?